crates/turbopack-ecmascript/src/lib.rs

```diff
@@ -84,13 +84,17 @@ use crate::{
 };
 
 #[turbo_tasks::value(serialization = "auto_for_input")]
-#[derive(PartialOrd, Ord, Hash, Debug, Default, Copy, Clone)]
+#[derive(PartialOrd, Ord, Hash, Debug, Copy, Clone)]
 pub struct EcmascriptOptions {
```

Comment:
We probably want to use a `EcmascriptOptionsBuilder` here to allow extending this in future (since it has no longer Default)